### DISCUSSION TITLE
Endless Sky Delta v0.10.9 parallel release

### DISCUSSION THREAD

**Discussion by Zitchas** [endless-sky/Endless-Sky-Delta#83] [General]
Greetings! Welcome to another release of Endless Sky Delta. Aside from the upstream changes, the major change of interest to content creators is probably the addition of support for limiting fleets and spawn numbers. This code provides the tools to enable a lot more control over how fleets spawn. A quick example is being able to set a fleet to a very fast rate of spawn to ensure it's always present. Previously (and in upstream) this would result in the fleet spamming very quickly and generating a lot of ships. With this code, however, it can also be given a limit to ensure that it does not exceed specific amounts. UnorderedSigh provided a demo file with working examples in the PR that I highly recommend content creators check out.

For players, I would just like to note that the number of previous saves has been bumped up slightly as the galaxy has gotten a bit bigger, and thus the risk of getting stranded is a little higher than it used to be. This should provide a bit more margin of safety.

One major change that everyone should be aware of is that as of this release, Endless Sky Delta now uses it's own folder for storing preferences, save files, etc. Depending on your operating system, this is referring to the folder located in AppData or its equivalents. Given the number of changes in the preference file between Delta and Upstream, we felt it made sense to actually separate the two so that people can switch back and forth without erasing settings for things like lateral thrust. Personally, I recommend copying over your preference file from the old Endless Sky folder just to save some work in setting up your preferences again. While preference files, plug-ins, and pilots are largely cross-compatible, as more changes happen the more it will be like switching between a present day version of ES and a much older version of ES. It will work, but risks losing some data and having some odd results here and there. Everything in upstream will be in Delta, but not everything in Delta will be in Upstream.

On the interesting side of things, Quantumshark revised Alpha Centauri to closely match the real-world equivalent system. This is particularly noteworthy since Alpha Centauri has three stars that orbit each other, and thus replicating that in Endless Sky means that the stars each appear to orbit an invisible central point instead of all orbiting one entity like Sol and other uni-stellar systems. Since some of the planets circle one star, and some another, it results in having two clusters of things in the system that are generally separated from each other. While only one can be landed on currently, it opens up all sorts of interesting possibilities for future content and entertainment.

## What's Changed
* Scanning for the Electronic Warfare System by @TheGiraffe3 in https://github.com/endless-sky/Endless-Sky-Delta/pull/71
* feat(data) Add Writers and Special Thanks sections to the credits. (also fixed folderization error from upstream) by @Zitchas in https://github.com/endless-sky/Endless-Sky-Delta/pull/76
* Use a new directory for saves, and bump the "previous saves" number to five by @TheGiraffe3 in https://github.com/endless-sky/Endless-Sky-Delta/pull/78
* feat(data): Expanding Alpha Centauri, and rearranging to be circumstellar rather than circumbinary by @Quantumshark in https://github.com/endless-sky/Endless-Sky-Delta/pull/73
* Limit number of fleets & spawn specified number when entering system by @UnorderedSigh in https://github.com/endless-sky/Endless-Sky-Delta/pull/24

## New Contributors
* @TheGiraffe3 made their first contribution in https://github.com/endless-sky/Endless-Sky-Delta/pull/71
* @Quantumshark made their first contribution in https://github.com/endless-sky/Endless-Sky-Delta/pull/73

**Full Changelog**: https://github.com/endless-sky/Endless-Sky-Delta/compare/v0.10.8-Delta...v0.10.9-Delta

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/Endless-Sky-Delta/releases/tag/v0.10.9-Delta'>Endless Sky Delta v0.10.9 parallel release</a>.</em>

**Comment by UnorderedSigh** [comment]
Now I'm one step closer to world domination, all thanks to you.

**Reply by Zitchas** [reply]
If world domination means improving the tools content creators have to create interesting, immersive, and well-balanced NPC fleet activities... Then it sounds good to me.

**Reply by UnorderedSigh** [reply]
I'm preparing to PR the Korath first contact PR (8752) so you should have that soon. Hopefully today, if I don't run into unexpected problems.

EDIT: That should say PR 8752
